src/components/confirm/confirm.style.js

```diff
@@ -1,11 +1,28 @@
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
+import PropTypes from 'prop-types';
 import confirmClassicStyle from './confirm.classic.style';
+import StyledIcon from '../icon/icon.style';
+import baseTheme from '../../style/themes/base';
 
-const StyledConfirmButtons = styled.div`
+export const StyledConfirmButtons = styled.div`
   margin-top: 48px;
   display: flex;
   justify-content: flex-end;
   ${confirmClassicStyle}
 `;
 
-export default StyledConfirmButtons;
+export const StyledInternalConfirm = styled.div`
```

Comment:
Incorrect component name, should be more descriptive like `StyledConfirmHeading` or `StyledHeadingContainer`